dev-lang/srf/srf-1.0-r2.ebuild

```diff
@@ -12,12 +12,12 @@ SRC_URI="http://www.stratigery.com/srf.html/${P}.tar.gz"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86 ~x86-linux ~ppc-macos"
-IUSE=""
 
 DEPEND="
 	sys-devel/flex
 	virtual/yacc"
-RDEPEND=""
+
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
 
 src_prepare() {
```

Comment:
`eutils` can go, and change the end of `src_install` to `dodoc -r examples` (and remove the docinto)

Reply:
Done :)